baselines/dom.generated.d.ts

```diff
@@ -19462,7 +19461,6 @@ interface SVGElementTagNameMap {
     "rect": SVGRectElement;
     "script": SVGScriptElement;
     "stop": SVGStopElement;
-    "style": SVGStyleElement;
```

Comment:
should this be deleted? It might be necessary to re-add it in addedTypes.json

Reply:
it is re-added